### PR TITLE
bpo-27682: Update whatsnew/3.8.

### PR DESCRIPTION
**Pull request by PetterS**
Update whatsnew/3.8.

https://bugs.python.org/issue27682

### CODE REVIEW

**Review by Carreau — APPROVED** [review]

File: Doc/whatsnew/3.8.rst [hunk 1]
```diff
@@ -748,6 +748,14 @@ weakref
   multiplication operators ``@`` and ``@=`` in addition to the other
   numeric operators. (Contributed by Mark Dickinson in :issue:`36669`.)
 
+wsgiref
+-------
+
+* Client disconnections and terminations no longer print long stack traces
+  to the terminal. This will make e.g. the Django development less noisy on
+  Windows. (Constributed by Berker Peksag in :issue:`29183` and Petter
```
Comment (mangrisano) [inline Doc/whatsnew/3.8.rst]: Maybe you meant "Contributed"? :)

Comment (PetterS) [reply Doc/whatsnew/3.8.rst]: Thanks

**Review by mangrisano — APPROVED** [review]

**Comment by PetterS** [comment]
Tagging @berkerpeksag since you are mentioned in the added text.

**Review by berkerpeksag — COMMENTED** [review]
Thank you for documenting these changes here. However, we usually document new features, breaking changes, deprecations etc. in What's New documents. Both issues, while quite annoying, were basically bug fixes and they can be found at https://docs.python.org/3.8/whatsnew/changelog.html.

**Comment by PetterS** [comment]
OK, let's close this then unless anyone objects.

**Pull request closed without merging**